jobserver/authorization/roles.py

```diff
@@ -22,18 +22,22 @@
 
 class CoreDeveloper:
     """
-    Bennett staff member with administrator access to Job Server.
+    Temporary vestigial CoreDeveloper role to allow migration
```

Comment:
I'm not sure I understand why this exists? Is it because the references weren't changed in the same commit, failing CI? If this is merged with the references commit, you don't need this class anymore I think as nothing references it?

Reply:
I've come to realise that this is actually necessary to make the migration run when there is `jobserver.authorization.roles.CoreDeveloper` role records in the db (see the updated PR description).

Reply:
This is no longer the case after making `parse_role()` more resilient